external/sql/storm-sql-core/src/jvm/org/apache/storm/sql/StormSqlImpl.java

```diff
@@ -166,8 +161,7 @@ public void explain(Iterable<String> statements) throws Exception {
         SqlNode validate = planner.validate(parse);
         RelNode tree = planner.convert(validate);
 
-        // TODO: change to all attributes when we change to cost-based planner
-        String plan = RelOptUtil.toString(tree, SqlExplainLevel.NON_COST_ATTRIBUTES);
+        String plan = StormRelUtils.explain(tree, SqlExplainLevel.ALL_ATTRIBUTES);
         System.out.println("plan>");
```

Comment:
should we move all these println statements to Utils print statement and in future we can change this to file based approach?